tests/AnnotationFinderTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+use Collective\Annotations\AnnotationFinder;
+use PHPUnit\Framework\TestCase;
+use Mockery as m;
+
+class AnnotationFinderTest extends TestCase
+{
+    /**
+     * @var m\LegacyMockInterface|m\MockInterface|Illuminate\Contracts\Foundation\Application
```

Comment:
Do we still really need `LegacyMockInterface`? Will it return that at all?

Reply:
I doubt it, that was just auto added based on the underlying code. I can take it out though